Makefile

```diff
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+SHELL := /bin/bash
+
```

Comment:
If we want to avoid forcing a bash dependency, I believe an alternative is to go with a subshell:
```
diff --git a/Makefile b/Makefile
index db4ec7ce..4d2ea398 100644
--- a/Makefile
+++ b/Makefile
@@ -12,8 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-SHELL := /bin/bash
-
 BUNDLE_VERSION = 4.5.1
 # OC_VERSION and BUNDLE_VERSION are going to same for release artifacts but
 # different for nightly and CI bits where bundle version would be any random
@@ -190,8 +188,8 @@ release: cross-lint embed_bundle build_docs_pdf gen_release_info
 	cd $(BUILD_DIR) && zip -r $(CURDIR)/$(RELEASE_DIR)/crc-windows-amd64.zip crc-windows-$(CRC_VERSION)-amd64
 
 	@mv $(RELEASE_INFO) $(RELEASE_DIR)/$(RELEASE_INFO)
-	
-	pushd $(RELEASE_DIR) && sha256sum * > sha256sum.txt && popd
+
+	( cd $(RELEASE_DIR) && sha256sum * > sha256sum.txt )
 
 HYPERKIT_BUNDLENAME = $(BUNDLE_DIR)/crc_hyperkit_$(BUNDLE_VERSION).$(BUNDLE_EXTENSION)
 HYPERV_BUNDLENAME = $(BUNDLE_DIR)/crc_hyperv_$(BUNDLE_VERSION).$(BUNDLE_EXTENSION)
```